plugins/gcc10/gcc10-overlay.mk

```diff
@@ -9,13 +9,12 @@ PKG             := cloog
 $(PKG)_TARGETS  := $(MXE_TARGETS)
 
 PKG             := binutils
-$(PKG)_VERSION  := 2.36.1
-$(PKG)_CHECKSUM := 5b4bd2e79e30ce8db0abd76dd2c2eae14a94ce212cfc59d3c37d23e24bc6d7a3
+$(PKG)_VERSION  := 2.37
+$(PKG)_CHECKSUM := 67fc1a4030d08ee877a4867d3dcab35828148f87e1fd05da6db585ed5a166bd4
```

Comment:
Put any gcc deps (binutils, gmp, mpfr, mpc) in a separate commit

Reply:
Sure. I will need to rewrite git history for this. So please keep it to this :)

Reply:
Fixed!